src/Parser/Hunter/Shared/Modules/Features/FocusChart/FocusTracker.js

```diff
@@ -81,7 +77,7 @@ class FocusTracker extends Analyzer {
   }
 
   checkActiveWaste(event) {
-    if ((event.sourceID === this.owner.player.id || event.targetID === this.owner.player.id) && event.classResources && event.classResources[0].type === RESOURCE_TYPES.FOCUS) {
+    if ((event.sourceID === this.owner.player.id || event.targetID === this.owner.player.id) && event.classResources && event.classResources[0].type === RESOURCE_TYPES.FOCUS.id) {
       this.tracker++;
       if (this.generatorCasts[event.ability.guid]) {
```

Comment:
Why does this line appear changed into this? It's already like this on live 🤔

Reply:
ye i think i fetched it a second before you fixed it, no idea why it gets highlighted tho